<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgHelpInfo</class>
 <widget class="QWidget" name="dlgHelpInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pBHome">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../ressourcen.qrc">
         <normaloff>:/img/images/go-home.png</normaloff>:/img/images/go-home.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBRefresh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>42</horstretch>
         <verstretch>30</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../ressourcen.qrc">
         <normaloff>:/img/images/edit-redo.png</normaloff>:/img/images/edit-redo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBBack">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../ressourcen.qrc">
         <normaloff>:/img/images/go-previous.png</normaloff>:/img/images/go-previous.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBNext">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../ressourcen.qrc">
         <normaloff>:/img/images/go-next.png</normaloff>:/img/images/go-next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lEUrl">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pBPrint">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../ressourcen.qrc">
         <normaloff>:/img/images/document-print.png</normaloff>:/img/images/document-print.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabPreview">
      <attribute name="title">
       <string>Preview</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabSource">
      <attribute name="title">
       <string>Sourcecode</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lUrl">
     <property name="text">
      <string>Url:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressourcen.qrc"/>
 </resources>
 <connections/>
</ui>
